django/db/backends/oracle/features.py

```diff
@@ -178,6 +178,10 @@ def supports_boolean_expr_in_select_clause(self):
     def supports_aggregation_over_interval_types(self):
         return self.connection.oracle_version >= (23,)
 
+    @cached_property
+    def supports_insert_multiple_rows(self):
```

Comment:
Wondering whether a better name might be one of:

- `supports_bulk_insert_with_table_values_constructor`
- `supports_bulk_insert_with_values`
- `supports_bulk_insert_with_multiple_rows`

? 🤔